### DISCUSSION TITLE
endless-sky would be cool if it was 3D

### DISCUSSION THREAD

**Discussion by Pantyhose-X** [endless-sky/endless-sky#8371] [Q&A] [answered]
endless-sky would be cool if it was 3D
Do you know [Limit Theory](https://en.wikipedia.org/wiki/Limit_Theory), which is a 3D implementation  https://www.youtube.com/watch?v=A2albJYS-wI

**Comment by RisingLeaf** [comment, marked as answer]
As much as 3D games are cool, it is not a development of Endless-Sky to be 3D, the contrary is the case, we define ourselves as a 2D space game. If you want a 3D space game you should search elsewhere.

That said there is some fanart that shows ships in 3D scenes:
https://www.youtube.com/watch?v=DIhDafOj95c

**Reply by Zireael07** [reply]
Yeah. OP should look at Oolite or Vega Strike imho

**Reply by SultanRTG** [reply]
Whatever happened to no sound in space? The animations is cood though.

**Comment by mOctave** [comment]
Honestly, this is something I've found myself thinking before too.

While Endless Sky itself would not make sense as a 2D game (as RisingLeaf said), most of the graphics are made in 3D, which means that it shouldn't be _too_ hard to make a version of it with most of the same content that exists in 3D. Additionally, almost all the assets are under the CC BY-SA 4.0 license, so if you wanted to you could make 3D Endless Sky with no legal issues.
